app/controllers/support_interface/settings_controller.rb

```diff
@@ -15,6 +15,7 @@ def deactivate_feature_flag
     end
 
     def notify_template
+      Site.last.dup.save! # Added as proof to test Sentry error handling
```

Comment:
Do you want to delete this before we approve? Just don't want to accidentally merge something in.

Another way to test, if you haven't already, is to deploy this branch to QA, sign into the Rails console and do:
```
begin
  1 / 0
rescue ZeroDivisionError => exception
  Sentry.capture_exception(exception)
end
```